README.md

```diff
@@ -36,9 +36,17 @@ Just create a project, and you’re good to go.
 
 Install it once globally:
 
-```sh
-npm install -g create-react-app
```

Comment:
The README mentions yarn alternatives for every other command, so I'd say the value is in consistency if nothing else.